core/src/test/scala/integration/kafka/api/ProducerSendWhileDeletionTest.scala

```diff
@@ -33,8 +33,8 @@ import scala.jdk.CollectionConverters._
 
 class ProducerSendWhileDeletionTest extends IntegrationTestHarness {
   val producerCount: Int = 1
-  val brokerCount: Int = 2
-  val defaultLingerMs: Int = 5;
+  val brokerCount: Int = 3
```

Comment:
Why does the tests need 3 brokers?

Reply:
I think due to the reassignment test? There may be a way to do it with just 2 though.

Reply:
as @jolshan mentioned this for easier test for reassignment